server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -1,24 +1,125 @@
 #!/bin/bash
 
+#
+# Deploy a self-contained Pbench server.
+#
+# This script creates a pod for the Pbench Server deployment using
+# four containers: the Pbench Server and its associated Elasticsearch
+# instance, PostgreSQL database instance, and KeyCloak broker
+# instance. After starting the dependent services, this script
+# configures the Keycloak broker, and then starts the Pbench Server.
+#
+# To invoke the pbench-server pod using this script it should run from the
+# root of the Git checkout.
+#
+# The resulting pod exports the following ports:
+#
+#   5432 - PostgreSQL
+#   8080 - HTTP web server (and Pbench Dashboard)
+#   8090 - keycloak authentication server
+#   9200 - Elasticsearch
+
 export PB_CONTAINER_REG=${PB_CONTAINER_REG:-"images.paas.redhat.com"}
-export PB_ORG_NAME=${PB_ORG_NAME:-"pbench"}
+export PB_CONTAINER_IMAGE_PREFIX="${PB_CONTAINER_REG}/pbench"
```

Comment:
This looks like a change from another PR.  Do we really want to make this change here?

Reply:
This change is from the latest commit and taken from Webb's comment [here](https://github.com/distributed-system-analysis/pbench/pull/3216#discussion_r1090917779)

Reply:
I suggested it in an earlier review.  And, yes, I think we should:  we don't use `PB_ORG_NAME` except in conjunction with the container registry name, so we might as well join them together and shorten the lines where it's referenced; and, I don't really think we need to parameterize the organization name.

Reply:
> and, I don't really think we need to parameterize the organization name.

I wonder about that ... I had a thought the other day trying to build a container for testing that it might have been nice to be able to build to `images.paas.redhat.com/dbutenho` ... and if anyone outside our group wants to pick this up, is hardcoding to "pbench" really a good idea?

(Note, it turns out that "local" build/deploy doesn't help me a bit because firing up a pbench pod on my laptop renders it useless from paging, which means I need a better laptop ... but in the meantime I'd need to push the image to somewhere from which I can pull it to a VM. Then again, my recent local attempts to build the container fail with an RPM dependency problem anyway... 😆 )

Reply:
> Wouldn't it have sufficed to build for `images.paas.redhat.com/pbench` with a _tag_ of `dbutenho`?

Yeah, and that'd be a fallback, although we might be cluttering our team repo with random private images. More importantly, should some other project decide to build a server (and that includes a CD pipeline for Red Hat managed services), a fixed "pbench" path might conflict with operational procedures; and configuring it from the beginning to be flexible isn't a bad thing.

> > it turns out that "local" build/deploy doesn't help me a bit because firing up a pbench pod on my laptop renders it useless from paging
> 
> It depends on your concept of "local". For me, my laptop is little more than a glorified "X Terminal",

Yeah, but I like to be able to work offline.

Reply:
Have we confirmed the removal of PB_ORG_NAME in all other places?

Reply:
Thanks for catching that Peter, there are still a few references left to be removed like in `runlocal` file and `pipeline.gy`.

Reply:
> Have we confirmed the removal of PB_ORG_NAME in all other places?

Let's not worry about removing them _all_ in **this** PR -- just address the ones which this PR affects.  (I think there is another PR from Peter, which I hope to supersede another from me, which removes the rest of them.)

Let's see if we can get this PR **_done_**....

Reply:
Or let's leave it the way it was and remove it in another PR, it was asked about 2 days ago.

Reply:
Or do you mean not to make a change in this file also?

Reply:
Okay, reverted these changes now